package.json

```
@@ -19,7 +19,8 @@
     "src",
     "src/index.d.ts"
   ],
-  "main": "src/index.js",
+  "main": "dist/node-mashov.min.js",
```

I'm not sure if it's better to have the minfied version as the entry point. The ["Authoring Libraries" guide](https://webpack.js.org/guides/author-libraries/#final-steps) uses the non-minified one.

```suggestion
  "main": "dist/node-mashov.js",
```